### PR TITLE
RDS Terraform completed

### PR DESCRIPTION
**Pull request by bentrzcinski**
- Adding the terraform folder and initialising it.
- Adding the terraform code for the RDS database instance
- Adding all the relevant extensions to the .gitignore file

Missing:
 - README.md
 
Note - The terraform requires some variables which will be stored locally to create the RDS, if you want to access the RDS yourself contact Ben to get the database username and password.

### CODE REVIEW

File: terraform/rds.tf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  performance_insights_enabled          = "true"
+  performance_insights_kms_key_id       = "arn:aws:kms:eu-west-2:129033205317:key/f62d672a-0743-4fd8-acdd-8a434d8fcb7c"
+  performance_insights_retention_period = "7"
+  port                                  = "5432"
+  publicly_accessible                   = "true"
+  skip_final_snapshot                   = "true"
+  storage_encrypted                     = "true"
+  storage_throughput                    = "0"
+  storage_type                          = "gp2"
+  vpc_security_group_ids                = ["sg-09cfa7d5cc78a795a"]
```
Comment (ebradley12) [inline terraform/rds.tf:38]: Maybe the security group ID should be in the .tfvars file for security

File: terraform/variables.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  description = "S3 database user name"
+  type        = string
+  sensitive   = true
+}
+
+variable "db_password" {
+  description = "password for the S3 database user"
+  type        = string
+  sensitive   = true
+}
```
Comment (ebradley12) [inline terraform/variables.tf:11]: Nice including a variables file

File: terraform/provider.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  region = "eu-west-2"
+}
+
+terraform {
+	required_providers {
+		aws = {
+	    version = "~> 5.78.0"
+		}
+  }
+}
```
Comment (ebradley12) [inline terraform/provider.tf:11]: I like this separate provide file.

File: terraform/rds.tf (hunk 1 above)
Comment (lukieh2014) [reply terraform/rds.tf]: +1

**Review by lukieh2014 — APPROVED** [review]
looks good to me, nice addition of .tfvars for security!

**Pull request merged**